src/assets/images/lockIconGrey.svg

```diff
@@ -0,0 +1,7 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="11" height="13" viewBox="0 0 11 13">
```

Comment:
I think it could be png as other icons, but let's keep it as an SVG if you did it

Reply:
arent svgs better for rendering?

Reply:
I don't think so there is any significant gain here